### PR TITLE
Example suggested change

### PR DESCRIPTION
**Pull request by azeemba**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # BSP-team1
 Website development
+
+# Adding Extra Section
```
Comment (azeemba) [inline README.md:4]: This looks good to me.

Comment (azeemba) [reply README.md:4]:
```suggestion
# Adding Section
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,4 @@
 # BSP-team1
```
Comment (azeemba) [inline README.md:1]: A lot of feedback. #1

File: README.md (hunk 1 above)
Comment (azeemba) [reply README.md:4]: More feedback. #2

**Review by dinaolmelak — APPROVED** [review]
Looks GooD : )

**Review by adebola-a — APPROVED** [review]

**Pull request closed without merging**